src/routes/status.tsx

```diff
@@ -6,21 +6,256 @@
+    color: string;
+    owner: string;
+    finalized: boolean;
+    priorityTimestamp: string;
+    stake: string;
+    0: string;
+    1: string;
+    2: string;
+    3: boolean;
+    4: string;
+    5: string;
+  }
+  suspect: boolean
+}
+
+@inject('exitHandler', 'explorer', 'web3', 'tokens')
 @observer
-export default class Status extends React.Component {
+export default class Status extends React.Component<StatusProps, {}> {
+  private exitColumns: ColumnProps<any>[] = [
+    {
+      title: 'Amount',
+      dataIndex: 'exit.amount',
+      key: 'amount',
+      render: (value, record) => (
+        <TokenValue value={value} color={record.exit.color} />
+      )
+    },
+    {
+      title: 'Owner',
+      dataIndex: 'exit.owner',
+      key: 'owner'
+    },
+    {
+      title: 'PriorityTimestamp',
+      dataIndex: 'exit.priorityTimestamp',
+      key: 'priorityTimestamp',
+      render: text => {
+        return <p>{new Date(parseInt(text) * 1000).toUTCString()}</p>;
+      }
+    },
+    {
+      title: 'Stake',
+      dataIndex: 'exit.stake',
+      key: 'stake',
+      render: (value, record) => (
+        <TokenValue value={value} color={record.exit.color} />
+      )
+    },
+    {
+      title: 'Suspicious',
+      dataIndex: 'suspect',
+      key: 'suspect',
+      render: suspect => {
+        if (suspect) return <p style={{ color: 'red' }}>Yes</p>;
+        else return <p style={{ color: 'green' }}>No</p>;
+      }
+    }
+  ];
+
+  private colorColums: ColumnProps<any>[] = [
+    {
+      title: 'Color',
+      dataIndex: 'color',
+      key: 'color'
+    },
+    {
+      title: 'Plasma balance',
+      dataIndex: 'plasmaBalance',
+      key: 'plasmaBalance',
+      render: (value, record) => (
+        <TokenValue value={value} color={record.color} />
+      )
+    },
+    {
+      title: 'Utxo Sum',
+      dataIndex: 'utxoSum',
+      key: 'utxoSum',
+      render: (value, record) => (
+        <TokenValue value={value} color={record.color} />
+      )
+    },
+    {
+      title: 'Exit Sum',
+      dataIndex: 'exitSum',
+      key: 'exitSum',
+      render: (value, record) => (
+        <TokenValue value={value} color={record.color} />
+      )
+    },
+    {
+      title: 'Status',
+      key: 'status',
+      render: (value, record: ColorDetails) => {
+        return record.plasmaBalance.gte(record.exitSum.add(record.utxoSum)) ? (
+          <Badge status='success' text='Ok' />
+        ) : (
+          <Badge status='warning' text='Invalid' />
+        );
+      }
+    }
+  ];
+
+  constructor(props) {
+    super(props);
+    reaction(() => props.exitHandler.address, this.getData)
+  }
+
+  @observable exitDetails: ExitData[] = [];
+  @observable colorDetails: ColorDetails[] = [];
+
+  @action
+  private getData = async (): Promise<void> => {
+    const { exitHandler, explorer, web3, tokens } = this.props;
+    const exits = []
+    const events = await exitHandler.contract.getPastEvents('ExitStarted', {fromBlock: 0})
```

Comment:
this will probably not work on main-net. better to start from deployment of contract like this: https://github.com/leapdao/leap-node/blob/master/src/bridgeState.js#L75-L77
and then go in batches like here: https://github.com/leapdao/leap-node/blob/master/src/utils/ContractsEventsSubscription.js